copydb/cmd/main.go

```diff
@@ -106,11 +106,13 @@ func main() {
 
 	err = ferry.Initialize()
 	if err != nil {
+		ferry.Ferry.ErrorHandler.ReportError("ferry.initialize", err)
```

Comment:
`ferry.ErrorHandler` gets initialized part way through in `ferry.Initialize` but there are several errors that could happen before it is initialized (ex.[ can't connect to source](https://github.com/Shopify/ghostferry/blob/b7ed73a3bcbeed243a92ed3b961d960e92afa309/ferry.go#L336-L340)). https://github.com/Shopify/ghostferry/blob/b7ed73a3bcbeed243a92ed3b961d960e92afa309/ferry.go#L425-L431 


We should initialize the error handler before to avoid this nil pointer exception.

Reply:
shuhao made a fix for this - https://github.com/Shopify/ghostferry/pull/276. I think the ghostferry version you have is an old one, pulling the latest master you will the changes.

Reply:
🤦 I approved that PR too.